setup.cfg

```
@@ -34,7 +34,7 @@ install_requires =
     pymysql~=0.9
     nglview~=3.0
     optimade-client==2022.9.19
-    spglib~=1.14
+    spglib~=2.0.2
```

I would do `~2.0`. Versions such as `2.1`, `2.2`, and `2.100` should still be ok.

```suggestion
    spglib~=2.0
```

---

Agree, I think in general we should not pin to specific versions, otherwise we'll be running into this issue over and over again.